test/fft.js

```diff
@@ -22,7 +20,7 @@ describe("FFT tests", () => {
 
         for (let i=0; i<N; i++) {
             const a = bn128.getInt(p+i*32);
-            assert.equal(a,i);
+            assert.equal(a.value,BigInt(i));
```

Comment:
does that work like it should? seems weird that it was passing before. Why can't it pass now without these changes?

Reply:
Previously, this assert was relying on JavaScript's implicit runtime type conversion to compare a bn128 object to a JS `number`.  Perhaps this worked in previous versions of NodeJS, but using `require('node:assert')` at least, that comparison will fail.  `bn128.value` returns a native `bigint` though, so we can just compare that to `i` as done here.